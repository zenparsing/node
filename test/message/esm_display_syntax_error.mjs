// Flags: --module
'use strict';
await async () => 0;
