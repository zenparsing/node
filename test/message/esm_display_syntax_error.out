file:///*/test/message/esm_display_syntax_error.mjs:3
await async () => 0;
^^^^^
SyntaxError: Unexpected reserved word
    at ModuleLoader._loadModule (internal/modules/esm/module_loader.js:*:*)
